internal/server/node/golden/get_property_labels_test.go

```diff
@@ -68,25 +68,27 @@ func TestGetPropertyLabels(t *testing.T) {
 				t.Errorf("could not GetPropertyLabels: %s", err)
 				continue
 			}
-			var result map[string]*model.PropLabelCache
-			err = json.Unmarshal([]byte(resp.GetPayload()), &result)
-			if err != nil {
+			// Here the golden file is not same as the actual API output.
```

Comment:
I don't fully follow this "data:{" hack.  I wonder if its worth documenting in the README clearly? (e.g., for someone fixing this code)

Reply:
Added more clarification of the case. This will be deprecated once we start on the v1 version of this API.